### PR TITLE
LGSVL Simulator change (part 4)

### PR DESCRIPTION
**Pull request by rongguodong**
Add lgsvl_msgs.

### CODE REVIEW

File: lgsvl_pkgs/lgsvl_msgs/CHANGELOG.rst [hunk 1]
```diff
@@ -0,0 +1,12 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (xiaoxq) [inline lgsvl_pkgs/lgsvl_msgs/CHANGELOG.rst:1]: As mentioned in https://github.com/ApolloAuto/apollo/pull/11187, we'd prefer to maintain such modules in `modules/contrib`.

File: lgsvl_pkgs/lgsvl_msgs/proto/detection3darray.proto [hunk 2]
```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+import "modules/common/proto/header.proto";
+import "lgsvl_pkgs/lgsvl_msgs/proto/detection3d.proto";
+
+package apollo.common;
```
Comment (xiaoxq) [inline lgsvl_pkgs/lgsvl_msgs/proto/detection3darray.proto:5]: Please use things like `apollo.contrib.lgsvl_pkgs`

File: lgsvl_pkgs/lgsvl_msgs/proto/BUILD [hunk 3]
```diff
@@ -0,0 +1,14 @@
+proto_library(
+    name = "lgsvl_lib",
+    srcs = [
+        "detection2d.proto",
+        "detection2darray.proto",
+        "detection3d.proto",
+        "detection3darray.proto",
```
Comment (xiaoxq) [inline lgsvl_pkgs/lgsvl_msgs/proto/BUILD:7]: As mentioned in https://github.com/ApolloAuto/apollo/pull/11187, prefer slim targets, with 1 proto in `srcs` of a library.

File: lgsvl_pkgs/lgsvl_msgs/CHANGELOG.rst (hunk 1 above)
Comment (rongguodong) [reply lgsvl_pkgs/lgsvl_msgs/CHANGELOG.rst]: Moved.

File: lgsvl_pkgs/lgsvl_msgs/proto/detection3darray.proto (hunk 2 above)
Comment (rongguodong) [reply lgsvl_pkgs/lgsvl_msgs/proto/detection3darray.proto]: Changed.

File: lgsvl_pkgs/lgsvl_msgs/proto/BUILD (hunk 3 above)
Comment (rongguodong) [reply lgsvl_pkgs/lgsvl_msgs/proto/BUILD]: Changed.

**Pull request merged**